glide.yaml

```diff
@@ -1,10 +1,11 @@
 package: github.com/opencontainers/image-spec
 import:
 - package: github.com/pkg/errors
-  version: ">=0.7.0"
+  version: '>=0.7.0'
 - package: github.com/xeipuuv/gojsonschema
   version: d5336c75940ef31c9ceeb0ae64cf92944bccb4ee
 - package: github.com/russross/blackfriday
   version: ~v1.4
 - package: github.com/shurcooL/sanitized_anchor_name
   version: 10ef21a441db47d8b13ebcc5fd2310f636973c77
+- package: github.com/opencontainers/go-digest
```

Comment:
No, it's here in glide.yaml - every other package has an associated version. Can we just keep it consistent for now? IMO glide-update should ideally be idempotent unless someone intentionally requests otherwise.